examples/printservices/src/main/java/org/icepdf/os/examples/print/PrintServices.java

```diff
@@ -18,7 +18,6 @@
 import org.icepdf.core.exceptions.PDFSecurityException;
 import org.icepdf.core.pobjects.Document;
 import org.icepdf.core.util.Defs;
-import org.icepdf.ri.common.PrintHelper;
```

Comment:
I think this might have been a mistake, can you update it to use the new factory.

Reply:
Yes I forgot to check the examples, good catch